enterprise/backend/test/metabase_enterprise/advanced_permissions/models/permissions_test.clj

```diff
@@ -50,40 +50,90 @@
           (is (nil? (download-perms-by-group-id group-id))))
 
         (testing "Download perms for individual tables can be set and revoked"
-          (let [[id-1 id-2 id-3 id-4] (map u/the-id (database/tables (mt/db)))]
+          (let [[id-1 id-2 id-3 id-4 id-5 id-6 id-7 id-8] (map u/the-id (database/tables (mt/db)))]
```

Comment:
Now that `sample-dataset` is moved into `test-data`, there are 8 ids (tables), not 4.